scripts/agent.sh

```diff
@@ -0,0 +1,23 @@
+if [[ $1 == 'install' ]]; then
```

Comment:
instead of install call it 'upgrade'

Reply:
upgrade `docker-compose pull && docker-compose up -d && sleep 30 && docker exec -i anodot-agent agent update` (sleep 30 -- to wait until streamsets is up)